src/gui/CMakeLists.txt

```diff
@@ -261,6 +261,8 @@ if(CMAKE_SYSTEM_NAME STREQUAL "FreeBSD")
   set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -L/usr/local/lib")
 endif()
 
+set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -DQT_QML_DEBUG ")
```

Comment:
I did mean to commit this as it corresponds to the debug lines on the qml files. Why shouldn't this flag be set (which takes only effect on Debug builds AFAIK?) to enable QML debugging just like the rest of the code base (that just doesn't need an extra flag bc it's c++. Why wouldn't you want to enable builtin QML debug functionality if you do a debug build?

Reply:
Because sometimes you might want to be able to distribute a debug build (as in having debug symbols shipped with the binary for easier crash analysis) without also having an open security risk distributed with it (and it's what the QML debugging facility is, it's basically opening a TCP socket waiting for connections and to receive JS code).

To make it more convenient for developers (seems it's what you're after) we could control it from a CMake option I guess? But unconditionally based on the build type is IMHO a security issue waiting to happen as mentioned above.

Reply:
Will remove it. As you said, you can as well refine and document used CMAKE options for possible contributors if they start to work with this codebase.